internal/querycoordv2/observers/collection_observer.go

```diff
@@ -162,9 +164,13 @@ func (ob *CollectionObserver) observeCollectionLoadStatus(collection *meta.Colle
 
 	segmentTargets := ob.targetMgr.GetHistoricalSegmentsByCollection(collection.GetCollectionID(), meta.NextTarget)
 	channelTargets := ob.targetMgr.GetDmChannelsByCollection(collection.GetCollectionID(), meta.NextTarget)
+	segmentTargetIDs := lo.MapToSlice(segmentTargets, func(k int64, _ *datapb.SegmentInfo) int64 {
+		return k
+	})
 	targetNum := len(segmentTargets) + len(channelTargets)
 	log.Info("collection targets",
 		zap.Int("segmentTargetNum", len(segmentTargets)),
+		zap.Int64s("segmentTargetIDs", segmentTargetIDs),
```

Comment:
As I known, target manager will output the segments while it updating the target, it's easy to find the segments list, and the collection observer runs per second, which may produce large logs for these lists

Reply:
Or output this list with another debug level log call.